docs/config.json

```diff
@@ -476,7 +476,19 @@
         {
           "title": "Access Graph",
           "slug": "/access-graph/",
-          "forScopes": ["team"]
+          "forScopes": [
+            "team",
+            "enterprise"
+          ],
+          "entries": [
+            {
+              "title": "Getting Started On-Premises",
+              "slug": "/access-graph/getting-started-on-prem/",
```

Comment:
It's a little counterintuitive to me that the `/access-graph` path is part of "Manage Access", but isn't part of the `/access-controls` path.

Reply:
Moved under `/access-controls/` in 35e9efd14b